tool/flutter_ci_script_master.sh

```diff
@@ -47,7 +47,8 @@ declare -ar PROJECT_NAMES=(
     "infinite_list"
     "ios_app_clip"
     "isolate_example"
-    "material_3_demo"
+#   TODO @ewindmill -- Color.red is deprecated and should be Color.r (same for green and blue)
+#    "material_3_demo"
```

Comment:
@jwill FYI, this is how to turn CI off for a specific sample in a specific environment.
We do this when an API is being deprecated in an upcoming version, and theres no way to make CI for Stable and beta channels happy at the same time. Ahead of every release, I update the sample code to reflect what will be in the new stable, and then add all the samples back to CI for the beta and master flutter channels.